src/components/bread-crumb/GroupBreadCrumbs.tsx

```diff
@@ -31,7 +32,7 @@ export const GroupBreadCrumbs = () => {
           </BreadcrumbItem>
           {subGroups.map((group, i) => (
             <BreadcrumbItem key={i} isActive={subGroups.length - 1 === i}>
```

Comment:
Perhaps we can use the group id for the key instead?
```suggestion
            <BreadcrumbItem key={group.id} isActive={subGroups.length - 1 === i}>
```